src/api/one/profiler/AsyncProfiler.java

```diff
@@ -271,5 +286,7 @@ private void filterThread(Thread thread, boolean enable) {
 
     private native String execute0(String command) throws IllegalArgumentException, IllegalStateException, IOException;
 
+    private native byte[] executeAndGetByteArray0(String command) throws IllegalArgumentException, IllegalStateException, IOException;
```

Comment:
I'm happy to hear alternatives for the name of this method, I could not think of anything better

Reply:
`execute1`?
Look, this is a private method, which does the same as `execute0` except for its return type - why should it be called differently? And yes, this is not uncommon: [1](https://github.com/openjdk/jdk/blob/2dfbf41d2a3dbcd44f9ed9a58a1b0932d7536977/src/java.base/share/classes/java/lang/ClassLoader.java#L1057-L1062), [2](https://github.com/openjdk/jdk/blob/2dfbf41d2a3dbcd44f9ed9a58a1b0932d7536977/src/java.management/share/classes/sun/management/ThreadImpl.java#L516-L521)